service/gradle/dependencies.gradle

```diff
@@ -8,6 +8,7 @@ dependencies {
 
   implementation 'com.google.cloud:google-cloud-bigquery'
   implementation 'com.google.cloud:google-cloud-bigquerydatatransfer'
+  implementation 'com.google.cloud:google-cloud-notebooks:1.10.0'
```

Comment:
I don't think we need the version here, unless the right version is not supplied by the google BOM.
The idea of the BOM is that all of the different libraries play nice with each other.